app/controlplane/extensions/core/template/v1/extension.go

```diff
@@ -178,10 +180,15 @@ func (i *Integration) Execute(_ context.Context, req *sdk.ExecutionRequest) erro
 // In this case we expect to receive a SBOM in CycloneDX format
 // plus registration, attachment state and credentials
 func validateExecuteRequest(req *sdk.ExecutionRequest) error {
-	if req == nil || req.Input == nil || req.Input.Material == nil || req.Input.Material.Content == nil {
-		return errors.New("invalid input")
+	if req == nil || req.Input == nil {
+		return errors.New("execution input not received")
+	}
+
+	if req.Input.Material == nil || req.Input.Material.Content == nil {
```

Comment:
We could add two use cases here:
- validation if material
- validation if attestation envelope